httpx/_client.py

```diff
@@ -418,8 +419,7 @@ class Client(BaseClient):
     URLs.
     * **timeout** - *(optional)* The timeout configuration to use when sending
     requests.
-    * **pool_limits** - *(optional)* The connection pool configuration to use
-    when determining the maximum number of concurrently open HTTP connections.
+    * **limits** - *(optional)* The limits configuration to use.
```

Comment:
Haven't found a better description of the parameter that's not redundant with the info in the `Limits` docstring. I don't think it's included in the Developer Interface yet so that could be a nice follow-up, and then cross-link it here?

Reply:
I guess we could say something like... "Allows for configuration options, such as the maximum number of open connections to allow." Just to give a bit of an indication what we mean by "limits" here(?)

Reply:
Let's pull this in for now as it is. Welcome to follow up with docstring improvements later.